bb-email/pom.xml

```diff
@@ -60,6 +64,11 @@
 
         <!-- tests -->
         <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
```

Comment:
The scope for this dependency is inherited from parent pom. Please remove it from here.
(should have)